### PR TITLE
Hw1

### PR DESCRIPTION
**Pull request by zfreiter**
Added solutions for all the problems.

- For problem two, I created a new element to append onto the last div.
- I added bootstrap classes with Javascript to add colors depending on the user input. 
- The solution also checks for classes that are added to the new element and updates them on the user input.

Looking forward to any feed back.

### CODE REVIEW

**Review by rutujapadgilwar — COMMENTED** [review]
01-selectors.md - Everything looks good!
02-palindrome.html - Your code looks good, nothing to improve.

File: hw1/02-palindrome.js [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+      }
+      para.classList.add('text-danger');
+    }
+  } else {
+    if (para.classList.contains('text.success')) {
+      para.classList.remove('text-success');
+    }
+    para.classList.add('text-danger');
+    para.innerHTML = 'Value must be a positive number.';
+  }
```
Comment (jpb7) [inline hw1/02-palindrome.js:44]: As far as I know, it's good practice to use classes to change the appearance of output messages; still, this `if-else` block seems avoidably verbose.

Something like the third example down on [this page](https://www.w3schools.com/jsref/prop_html_classname.asp) might offer a slimmer, more direct alternative.

**Review by jpb7 — COMMENTED** [review]
The Bootstrap looks good to me. I'm not sure whether the card is supposed to stretch with different screen sizes: in any case, it fits legibly on all the dimensions I tried. You might consider refactoring the `if-else` block of your JavaScript to make it more compact. It definitely works for all the cases I tried, but I can't help thinking you could accomplish the same thing with fewer operations.

File: hw1/02-palindrome.js (hunk 1 above)
Comment (zfreiter) [reply hw1/02-palindrome.js:44]: I thought I had multiply classes on the para element so I was making sure that I was only removing the desired class. I recheck and notice that I was only added the bootstrap color class and cleaned up the if else.

**Review by ssiddhi — COMMENTED** [review]
Hi, This is the feedback on your code from my side.

**Pull request merged**